src/command.c

```diff
@@ -163,7 +163,7 @@ void dv_cmb_finalize(struct device *dev, struct dmp_cmb *cmb)
 
 	list_for_each_entry_safe(lobj, ltmp, &cmb->cmb_list, list_node) {
 		dma_free_coherent(dev, lobj->capacity, lobj->logical,
-			lobj->physical);
+				lobj->physical);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket